### PR TITLE
Address February 2018 WGLC comments

### PR DESCRIPTION
**Pull request by selfissued**

### CODE REVIEW

File: draft-ietf-secevent-token.xml [hunk 1]
```diff
@@ -164,27 +164,27 @@
...
+            some distribution method. A SET recipient is the same
+            entity referred as a "recipient" in <xref target="RFC7519"/> or "receiver" in
             related specifications.
           </t>
             
           <t hangText="Subject">
 	    <vspace/>
             A SET describes an event or state change that has occurred 
-            about a Subject. A Subject might, for instance, be a principal (e.g., 
+            about a subject. A subject might, for instance, be a principal (e.g.,
```
Comment (independentid) [inline draft-ietf-secevent-token.xml]: Subject is a defined term I believe. It should not be lowercased as it is in sec 1.5

**Comment by selfissued** [comment]
The RFC Editor's style guide will have us lowercase all the uses of the defined terms in the document, other than the defining one.  Annabelle was right to point out that our case usage was inconsistent.  This PR fixes that by doing what the RFC Editor would have us do anyway.

(For instance, that's why, even though "Authorization Server" is a defined term in OAuth, all uses of it other than the defining usage are spelled "authorization server".)

**Comment by independentid** [comment]
Are you sure? In my experience, the RFC Editor forces all defined terms to be capitalized so that the reader knows it is not a generic or english "subject" for example.

**Comment by selfissued** [comment]
I'm certain.  When we took text that came from OpenID specs, which did capitalize uses Defined Terms, and put them in RFCs, the RFC editor made us lowercase them when used.

File: draft-ietf-secevent-token.txt [hunk 2]
```diff
@@ -486,95 +542,101 @@ Internet-Draft          draft-ietf-secevent-token          February 2018
...
+      name/value pairs whose names are URIs identifying the event
       statements being expressed.  Event identifiers SHOULD be stable
       values (e.g., a permanent URL for an event specification).  For
       each name present, the corresponding value MUST be a JSON object.
       The JSON object MAY be an empty object ("{}"), or it MAY be a JSON
-      object containing data described by the Profiling Specification.
+      object containing data described by the profiling specification.
 
-   txn
+   "txn" (Transaction Identifier) Claim
```
Comment (independentid) [inline draft-ietf-secevent-token.txt:633]: I wonder if Yaron's concerns might be better addressed if it said "in which multiple related JWTs or SETs are issued".  I can go either way...you choose.

**Pull request merged**